gammapy/data/counts.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function, division
+from gammapy.spectrum import EnergyBinning
+
+
+__all__ = ['CountsSpectrumDataset',
+           ]
+
+
+class CountsSpectrumDataset(object):
+    """
+    Counts spectrum dataset (PHA wrapper).
+    
+    Test case for EnergyBinning class
```

Comment:
Do you want to implement a first version of this class in this PR?
If yes, add it to the task list and remove this comment.